pennylane_qiskit/converter.py

```diff
@@ -36,17 +36,83 @@
 dagger_map = {"SdgGate": qml.S, "TdgGate": qml.T, "SXdgGate": qml.SX}
 
 
-def _check_parameter_bound(param: Parameter, var_ref_map: Dict[Parameter, Any]):
+def _check_parameter_bound(param: Parameter, trainable_params: Dict[Parameter, Any]):
     """Utility function determining if a certain parameter in a QuantumCircuit has
     been bound.
 
     Args:
         param (qiskit.circuit.Parameter): the parameter to be checked
-        var_ref_map (dict[qiskit.circuit.Parameter, Any]):
+        trainable_params (dict[qiskit.circuit.Parameter, Any]):
```

Comment:
Seems like a pre-existing naming choice, but `trainable_params` does already have a connotation of `QuantumScript.trainable_params`, which is a different kind of thing.

Reply:
renamed to `unbound_params`